app/workers/commit_monitor_handlers/commit_range/rubocop_checker.rb

```diff
@@ -33,13 +33,31 @@ def process_branch
     nil # Avoid working on unmergeable PRs
   end
 
+  # requirements to "create_status(repo, sha, state, options)"
+  # state is a boolean variable describing success/error state
+  def commit_status(state)
+    {
+      "repo"    => fq_repo_name,
+      "sha"     => commits.last,
+      "state"   => state ? "success" : "error",
+      "options" => {
+        "context"     => "miq-bot", # TODO: it should be user name variable
+        "target_url"  => nil,
```

Comment:
Why is this `nil`? Can't be this parametrized as `state`?

Reply:
@skateman The key is required and the value is added when the URL of the comment is obtained.

Reply:
OK, what about the 2nd part of the question? :wink: